src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
 }
+
+:root {
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.logo {
+  cursor: pointer;
+  width: 40px;
+  height: 40px;
+  padding-top: 10px;
+  padding-left: 50px;
+}
+
+.nav {
+  margin-left: 10px;
+}
```

Comment:
redundant block

Reply:
i put padding: 0 in body and deleted redundant .nav
Tests are still failing though...